src/librustdoc/html/static_files.rs

```diff
@@ -64,9 +61,11 @@ pub(crate) fn static_filename(filename: &str, contents: &[u8]) -> PathBuf {
 }
 
 fn static_suffix(bytes: &[u8]) -> String {
-    let mut hasher = FxHasher::default();
-    hasher.write(bytes);
-    format!("-{:016x}", hasher.finish())
+    use sha2::Digest;
+    let bytes = sha2::Sha256::digest(bytes);
+    let mut digest = format!("-{bytes:x}");
+    digest.truncate(17);
```

Comment:
The original included the entire hash, but it's actually unusually long for a cachebuster. Why not

```suggestion
    digest.truncate(9);
```

Reply:
No particular reason. The full 256bit output was comicly-long, so I decided to shorten to the same length as the old scheme.